knowledge-base/adr/0021-odata-url-builder.md

```diff
@@ -0,0 +1,193 @@
+# Title
```

Comment:
Next time, please consider using the template in this directory, so that the documents always have a similar structure.

Reply:
No, this time. I merged the template;)

Reply:
Also nice 😊